src/Property/CSSNamespace.php

```diff
@@ -35,11 +37,10 @@ class CSSNamespace implements AtRule
     protected $comments = [];
 
     /**
-     * @param string $url
-     * @param string|null $prefix
+     * @param CSSString|URL $url
      * @param int<0, max> $lineNumber
      */
-    public function __construct($url, $prefix = null, int $lineNumber = 0)
+    public function __construct($url, ?string $prefix = null, int $lineNumber = 0)
     {
         $this->url = $url;
         $this->prefix = $prefix;
```

Comment:
The PHPDoc type annotation for the property should probably also be changed to `string|null` while we're here.